superset-frontend/src/dashboard/components/Header/index.jsx

```diff
@@ -156,7 +157,7 @@ class Header extends React.PureComponent {
     this.hidePropertiesModal = this.hidePropertiesModal.bind(this);
     this.showReportModal = this.showReportModal.bind(this);
     this.hideReportModal = this.hideReportModal.bind(this);
-    this.handleReportClick = this.handleReportClick.bind(this);
+    this.reportModal = this.reportModal.bind(this);
```

Comment:
can we make this more "action/verb" sounding for a method, like renderReportModal, or else we can make it a separate component outside the header (could be the same file)

Reply:
yeah I wasn't happy with this and couldn't place my finger on it, I had spaced on using render, sorry.